README.md

```diff
@@ -234,6 +243,7 @@ The easiest way to get up and running is to use the provided docker compose work
 1. Run tests: `docker compose run base rails test` The app has comprehensive test coverage but note that system tests currently do not work in docker.
 1. Open the rails console: `docker compose run base rails console`
 1. Run a psql console: `docker compose run base psql`
+1. Load fixtures: `docker compose run base rails db:fixtures:load`
```

Comment:
Let's give a bit more context as to why someone should run this command. Also, let's add this same line (without the "docker compose run base" prefix) to the section below. The very next section walks through setting up outside of docker and it can be helpful to have this step there too.

```suggestion
1. If you want a few fake users and a bunch of conversations and other data pre-populated in the database, you can load fixtures into the development database. This can be helpful, for example, if you want to test a migration and save yourself the time manually creating a bunch of data: `docker compose run base rails db:fixtures:load`
```